userscript/twitchnosub.user.js

```diff
@@ -34,15 +33,16 @@
 
     window.Worker = class Worker extends oldWorker {
         constructor() {
-            super(URL.createObjectURL(new Blob(["importScripts('https://cdn.jsdelivr.net/gh/besuper/TwitchNoSub/src/patch_amazonworker.js', 'https://cdn.jsdelivr.net/npm/amazon-ivs-player/dist/assets/amazon-ivs-worker.min.js');"])));
+            const blobUrl = URL.createObjectURL(new Blob([`importScripts('https://cdn.jsdelivr.net/gh/besuper/TwitchNoSub/src/patch_amazonworker.js', 'https://cdn.jsdelivr.net/npm/amazon-ivs-player/dist/assets/amazon-ivs-worker.min.js');`],));
+
+            super(blobUrl);
 
             if (!isVariantA) {
                 this.addEventListener("message", (event) => {
                     const data = event.data;
 
                     if (data.id == 1 && data.type == 1) {
-                        const newData = event.data;
-
+                        const newData = { ...data };
```

Comment:
is it really needed to use the spread operator here ? why `const newData = event.data;` does not work ?

Reply:
`const newData = { ...data };` avoids changing the original data.
 `const newData = event.data;` is causing `Error #4000`, Issue #144